packages/labs/examples/multiSelectExample.tsx

```diff
@@ -19,7 +19,7 @@ const INTENTS = [Intent.NONE, Intent.PRIMARY, Intent.SUCCESS, Intent.DANGER, Int
 
 export interface IMultiSelectExampleState {
     films?: Film[];
-    initialRenderer?: boolean;
+    initialContent?: boolean;
```

Comment:
i'd call this `hasInitialContent` to distinguish from the actual prop since this one's a boolean